### ISSUE TITLE
Quazar can't be spawned on a server

### ISSUE THREAD

**Issue by TCreopargh** [Rebirth-of-the-Night/Better-Slimes#6]
Looks like quazar can't be spawned on server environment. CraftTweaker script, commands, and spawn eggs all don't work.
This is the error printed in logs when doing this:

Note: quazar works fine in singleplayer
```
[13:17:07] [Server thread/ERROR] [FML]: Encountered an exception while constructing entity 'betterslimes:quazar'
java.lang.reflect.InvocationTargetException: null
        at sun.reflect.GeneratedConstructorAccessor272.newInstance(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_282]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_282]
        at net.minecraftforge.fml.common.registry.EntityEntryBuilder$ConstructorFactory.apply(EntityEntryBuilder.java:305) [EntityEntryBuilder$ConstructorFactory.class:?]
        at net.minecraftforge.fml.common.registry.EntityEntryBuilder$ConstructorFactory.apply(EntityEntryBuilder.java:291) [EntityEntryBuilder$ConstructorFactory.class:?]
        at net.minecraftforge.fml.common.registry.EntityEntry.newInstance(EntityEntry.java:68) [EntityEntry.class:?]
        at net.minecraft.entity.EntityList.createEntityByIDFromName(EntityList.java:204) [vi.class:?]
        at net.minecraft.entity.EntityList.createEntityFromNBT(EntityList.java:211) [vi.class:?]
        at net.minecraft.world.chunk.storage.AnvilChunkLoader.createEntityFromNBT(AnvilChunkLoader.java:602) [aye.class:?]
        at net.minecraft.world.chunk.storage.AnvilChunkLoader.readWorldEntityPos(AnvilChunkLoader.java:561) [aye.class:?]
        at net.minecraft.command.server.CommandSummon.execute(SourceFile:87) [ds.class:?]
        at com.feed_the_beast.ftbutilities.ranks.CommandOverride.execute(CommandOverride.java:83) [CommandOverride.class:?]
        at net.minecraft.command.CommandHandler.tryExecute(CommandHandler.java:119) [bj.class:?]
        at net.minecraft.command.CommandHandler.executeCommand(CommandHandler.java:91) [bj.class:?]
        at net.minecraft.network.NetHandlerPlayServer.handleSlashCommand(NetHandlerPlayServer.java:962) [pa.class:?]
        at net.minecraft.network.NetHandlerPlayServer.processChatMessage(NetHandlerPlayServer.java:941) [pa.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.processPacket(SourceFile:37) [la.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.processPacket(SourceFile:9) [la.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [hv$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_282]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_282]
        at net.minecraft.util.Util.runTask(SourceFile:529) [h.class:?]
        at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:723) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:397) [nz.class:?]
        at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:823) [?:1.8.0_282]
Caused by: java.lang.NoSuchMethodError: net/minecraft/entity/EntityList.func_192839_a(Ljava/lang/String;)Ljava/lang/Class; (loaded from jar:file:/G:/GreedyCraft-Server/minecraft_server.1.12.2.jar!/vi.class by net.minecraft.launchwrapper.LaunchClassLoader@a94593a) called from class com.mic.betterslimes.entity.slimes.Quazar (loaded from jar:file:/G:/GreedyCraft-Server/./mods/BetterSlimes-1.12.2-1.3.1.jar!/com/mic/betterslimes/entity/slimes/Quazar.class by net.minecraft.launchwrapper.LaunchClassLoader@a94593a).
        at com.mic.betterslimes.entity.slimes.Quazar.<init>(Quazar.java:129) ~[Quazar.class:?]
        ... 27 more
[13:17:07] [Server thread/WARN] [minecraft/EntityList]: Skipping Entity with id betterslimes:quazar
```

**Comment by democat3457** [comment]
oh so THATS why it doesnt work on servers... ;-;

**Comment by democat3457** [comment]
Fixed in version 1.3.2
